runtime/near-vm-runner/src/prepare.rs

```diff
@@ -223,11 +221,10 @@ impl<'a> ContractModule<'a> {
 
             let type_idx = match *import.external() {
                 External::Function(ref type_idx) => type_idx,
-                External::Memory(ref memory_type) => {
-                    imported_mem_type = Some(memory_type);
-                    continue;
-                }
+                External::Memory(_) => return Err(PrepareError::Memory),
                 _ => continue,
+                // TODO: it’s a protocol change because the gas amount burned would change for some tests, but here we should
+                // _ => return Err(PrepareError::Instantiate),
```

Comment:
Allowing imports of globals etc. seems fine to me, even if we don’t have any currently. As long as we confirm that they must come from `env` which we do, it is fine.

Reply:
Hmm so I was coming from the idea that right now we validate func types again (though I’m not really sure why), but we just ignore globals etc. Anyway it doesn’t change anything, so I’m removing this comment :)